src/Ben.Demystifier/Ben.Demystifier.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net45;net6</TargetFrameworks>
```

Comment:
Should this be `net6.0`? Or this really picks up any `net6.x`?

Reply:
Should really be

Reply:
Is fixed

Reply:
sorry about the extra work